models/evc.py

```diff
@@ -1757,6 +1757,13 @@ def handle_link_up(self, link):
                 lambda me: me.primary_path.is_affected_by_link(link),
                 lambda me: (me.deploy_to_primary_path(), 'redeploy')
             ),
+            # For this special case, it reached this point because interface
+            # was previously confirmed to be a UNI and both UNI are UP
+            (
+                lambda me: (me.primary_path.status == EntityStatus.UP
```

Comment:
@Alopalao dynamic paths wouldn't suffer from this same problem? I mean: a dynamic path which is not active and received an interface link UP that could be used to deploy that EVC. My point is: having this specific for primary_path won't leave room for problems with dynamic paths? or even EVCs in backup paths?

Reply:
I missed `backup_path`, added it.
When `primary_path` is empty, its status is `DISABLED`, it is the same for `backup_path`. So if the EVC is dynamic this condition will be `False` and try the next one. I am going to keep testing though,

Reply:
Good catch @italovalcy , indeed it was a corner case that wasn't properly handled for static EVCs when UNI tries to `try_to_handle_uni_as_link_up`. @Alopalao , you're in the right path, I think we'll need these predicate and actions here, parametring `interface` looks good. Yes, for dynamic EVCs [this predicate](https://github.com/kytos-ng/mef_eline/blob/master/models/evc.py#L1779) would be true and would case the expected redeploy. I managed to find out how to reproduce with a similar outcome, check out my comment below, see if it matches what you saw in the prod logs.

Reply:
Cool @Alopalao, let's also add a new e2e to cover the original issue, when it's ready to land. 

To make [the steps mentioned here](https://github.com/kytos-ng/mef_eline/pull/608#pullrequestreview-2564458471) as a new e2e test shouldn't too difficult